src/Test/Utilities/Portable/TargetFrameworkUtil.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using static TestReferences;
+
+namespace Roslyn.Test.Utilities
+{
+    public enum TargetFramework
+    {
+        /// <summary>
+        /// Explicit pick a target framework that has no references
+        /// </summary>
+        None,
+
+        NetStandard20,
+        WinRT,
+
+        /// <summary>
+        /// Eventually this will be deleted and replaced with NetStandard20. Short term this creates the "standard"
+        /// API set across destkop and coreclr 
+        /// </summary>
+        Standard,
+
+        /// <summary>
+        /// This is represents the set of tests which must be mscorlib40 on desktop but full net standard on coreclr.
+        /// </summary>
+        StandardCompat,
+
+        // The flavors of mscorlib we support + extending them with LINQ and dynamic.
+        Mscorlib40,
+        Mscorlib40Extended, 
+        Mscorlib45,
+        Mscorlib45Extended,
+        Mscorlib46,
+        Mscorlib46Extended,
+    }
+
+    public static class TargetFrameworkUtil
+    {
+        public static readonly ImmutableArray<MetadataReference> NoneReferences = ImmutableArray<MetadataReference>.Empty;
```

Comment:
Can we get rid of this field and just inline `ImmutableArray.Empty`? I think the devotion to consistency in this case is unnecessary and harms readability.

Also, none pizza with left beef: http://i0.kym-cdn.com/photos/images/newsfeed/000/838/967/e39.png

Reply:
Sure